docs/configuration/introduction.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: default
+title: Introduction
+parent: Advanced Configuration
+nav_order: 1
+has_children: false
+---
+
+## Introduction
+
+You can customize the default configuration by writing a JSON file and adding it to the library. Take a look at the following sections to get a better understanding of what you can configure and how:
+
+* [Configuration of logging fields](/cf-nodejs-logging-support/configuration/fields/)
+* [Default request logging level](/cf-nodejs-logging-support/configuration/defaultrequestlevel/)
+* [Custom fields format](/cf-nodejs-logging-support/configuration/customfieldsformat/)
+* [Framework](/cf-nodejs-logging-support/configuration/framework/)
+
+## Add custom configuration
+
+Once you have a JSON file with your configuration, you can add it to the logger as follows:
+
+```ts
+import configFile from './config.json';
+
+log.addConfig(configFile);
+```
+
+Alternatively you can set your configuration into differentes JSON files and add them using the same function.
+
+```ts
+import configFile1 from './config1.json';
+import configFile2 from './config2.json';
+import configFile3 from './config3.json';
+
+log.addConfig(configFile1, configFile2, configFile3);
+```
+
+Notice that in this case, the configuration files will be added interatively. This means that in case of collisions, the latest configuration file will override the previous one.
```

Comment:
Configuration files will be added iteratively. This means that in case of collisions latter configuration files will override previous ones.